ide-support/revliburl.livecodescript

```diff
@@ -664,7 +664,11 @@ on ulHostNameToAddressCallback, pHostname, pIP
    end if
    
    if tBadAddress then
-      put "invalid host address" into laUrlErrorStatus[tURL]
+      if the paramCount >= 3 then
+         put param(3) into laUrlErrorStatus[tURL]
```

Comment:
If you make the optional parameter explicit in the definition of the handler, then you can use `pOptionalError` instead of `param(3)` here.